app/controllers/cables_controller.rb

```diff
@@ -3,6 +3,10 @@
 class CablesController < ApplicationController
   before_action :set_cable, only: [:destroy]
 
+  def index
+    @cables = sorted Cable.includes(:connections, connections: [:port]).all
```

Comment:
```suggestion
    @cables = sorted Cable.includes(:connections, connections: [:port])
```

`.all` is not necessary. And add pagination like done for `ChangelogEntry`